src/lib/transaction/operations/allowTrust/AllowTrust.svelte

```diff
@@ -0,0 +1,23 @@
+<script lang="ts">
+    import type { TrustLineFlag } from 'stellar-sdk';
+
+    export let optionalSource: string | undefined;
+    export let defaultSource: string;
+    export let assetCode: string;
+    export let authorize: boolean | TrustLineFlag | undefined;
+</script>
+
+<div class="simple-signer allow-trust-operation">
+    <h3>Operation: {authorize === 0 || false ? 'Disallow' : 'Allow'} Trust</h3>
```

Comment:
Does this 0 not have a counterpart in TrustLineFlag?

Reply:
also, you are simply comparing `0 || false` this is not doing what you think it's doing